app/components/Footer.tsx

```diff
@@ -1,43 +1,105 @@
-import { MarkGithubIcon } from "@primer/octicons-react";
 import type { FC } from "react";
-import { DISCORD_LINK, REPO_URL } from "~/utils/constants";
+import { Link } from "@remix-run/react";
+import CallToAction from "~/components/CallToAction";
+import {
+  HOW_TO_LIST_PROJECT_LINK,
+  DISCORD_LINK,
+  REPO_LINK,
+  RESOURCES_LINK,
+  // SIGNIN_LINK,
+  // SIGNUP_LINK,
+  LINK_FORMATS,
+  ABOUR_LINK,
+} from "~/utils/constants";
 import CodeSeeWordmark from "~/images/CodeSeeWordmark";
-import DiscordIcon from "./icons/DiscordIcon";
+import logo from "~/images/logo.png";
+
+import gradientStyles from "~/styles/gradient.css";
+
+export function links() {
+  return [{ rel: "stylesheet", href: gradientStyles }];
+}
```

Comment:
**issue:** this won't work the way you expect because the Footer component isn't a route layer boundary (see my other comment on the index.tsx file).